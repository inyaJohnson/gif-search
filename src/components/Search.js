import React from 'react'
import { Row, Col, Form, InputGroup, Button } from 'react-bootstrap';

const Search = (props) => {
    const { searchWord, searchWordChange, searchWordSubmit } = props
    return (
        <Row>
            <Col xs={{ span:12, offset:0 }} md={{ span:10, offset:1 }} className='search-input'>
                <Form className='search-form' onSubmit={searchWordSubmit}>
                    <InputGroup>
                        <Form.Control size='lg' type='text' value={searchWord} onChange={searchWordChange} placeholder='Search for GIF' className='search-input-field'/> 
                        <Button className='searchBtn' variant='primary' type='submit'>Search</Button>
                    </InputGroup>
                </Form>
            </Col>
        </Row>
    )
}

export default Search;