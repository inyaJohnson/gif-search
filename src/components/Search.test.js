import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16'
import Search from './Search';

Enzyme.configure({ adapter: new Adapter() })

test('Search field box renders', () => {

  const searchBox = shallow(<Search />);

  expect(searchBox).toMatchSnapshot();
});