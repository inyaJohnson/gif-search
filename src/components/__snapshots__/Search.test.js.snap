// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search field box renders 1`] = `ShallowWrapper {}`;
